src/plugins/unified_search/public/filter_bar/filter_item/filter_item.tsx

```diff
@@ -379,18 +377,6 @@ export function FilterItem(props: FilterItemProps) {
     panelPaddingSize: 'none',
   };
 
-  if (props.readonly) {
```

Comment:
Was this the only usage of `readonly`? If so can we remove it from the props?

Reply:
Yes, I've already removed it.